br-ext/board/stmicroelectronics/stm32mp1-tz/post-image.sh

```diff
@@ -29,7 +29,9 @@ BOOTFS_DIR=${BASE_DIR}/target-bootfs
 rm -f ${BINARIES_DIR}/bootfs.ext2 || exit 1
 rm -rf ${BOOTFS_DIR} && mkdir -p ${BOOTFS_DIR}/boot || exit 1
 cp --dereference ${3}/uImage ${BOOTFS_DIR}/boot || exit 1
-cp --dereference ${3}/*.dtb ${BOOTFS_DIR}/boot || exit 1
+for f in ${3}/*.dtb; do
+	test -f $f && { cp --dereference  $f ${BOOTFS_DIR}/boot || exit 1; }
```

Comment:
If `$f` is a regular file, why use `--dereference`?

Reply:
Regular files can be symbolic links. We do create symlinks in **out/bin/**.
Here `test -f $f` ensures the symbolic link is not an invalid symlink from a old build.

Reply:
Ah, you're right, with a symlink to a regular file both `test -f` and `test -h` return true.